src/dragonball/src/api/v1/vmm_action.rs

```diff
@@ -760,6 +817,16 @@ impl VmmService {
     }
 }
 
+impl std::fmt::Debug for VmmService {
```

Comment:
Try to use this: https://mcarton.github.io/rust-derivative/latest/Debug.html

Reply:
same below

Reply:
Lesson learned. I use it for `TraceInfo`. But I delete the Debug trait for `VmmService` since we don't need to output the information of `VmmService` in tracing.